### PR TITLE
Portfolio Mobile Version Skeleton

### PR DESCRIPTION
**Pull request by leonardoeng13**
That's my first attempt to accomplish the figma template #5

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+  left: 160px;
+  top: 572px;
+}
+
+/* Menu animation */
+
+/* .desktop-nav {
+  animation: fadeDown 1000ms forwards;
+  opacity: 0;
+} */
```
Comment (Shaher-11) [inline style.css]: Delete the comment

File: style.css [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+  to {
+    opacity: 1;
+  }
+}
+
+/* .hero-container .left h1 {
+  animation: crossScreen 800ms forwards;
+  animation-delay: 800ms;
+  opacity: 0;
+} */
```
Comment (Shaher-11) [inline style.css]: Delete the comment

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @leonardoeng13,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9).
- [ ] The below images to show what is done so far and what the actual design is, so kindly style your project as shown in the original design.
#### Original Design
right
#### Your Project
wrong

- [ ] Update the readme file with the project screenshot and info as well as a live demo.
- [ ] Check the comments below:point_down:


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 reviews remaining. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Comment by SpaYco** [comment]
Hello 👋,

This is an additional review as a result of the second opinion request.

As you said in the request, these projects are made so you apply everything you learn and discover new things that you may not know, and force you to be able to make great designs like this! (also, check the [big picture](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/sneak_peek.md))

And for those reasons, unfortunately, you need to implement the changes asked by the reviewer, please read more about [flexbox](https://github.com/microverseinc/curriculum-html-css/blob/main/flexbox.md) so you be able to align the elements well on your page!

Remember that if you're stuck, you can always [ask your questions on our Discourse](https://questions.microverse.org/) or [schedule a meeting with a Senior Code Reviewer](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-#)!

Keep rocking 🚀!

File: style.css (hunk 1 above)
Comment (leonardoeng13) [reply style.css]: Removed

File: style.css (hunk 2 above)
Comment (leonardoeng13) [reply style.css]: Removed

**Comment by leonardoeng13** [comment]
Thanks all. Forcing me to look for more improvements, now I'm more confident about the program and my new skills.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @leonardoeng13  👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but we still a tiny issue left 🙌 , I don't mean to drag you back but you know it would be nice to learn the right skills.

 ## Required Changes ♻️
1. Kindly update your README file with a live demo link;

2. Your app still looks a little bit different, I hope you pushed your last codebase in your code editor, your app is still not mobile responsive;
Kindly watch the screencast from your app below;
mobile-design

Hint: It would be nice to use [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp).
- Header tag for the top part.
- Section tag for the middle content
- Section tag for the bottom content
Then, after the HTML Semantic structure, re-Flex as you're already using it.

_And if you're still stuck, it would be nice to set up a meeting with the Senior Code Reviewer as mentioned above_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
@leonardoeng13 I'm so sorry for the mix-up 🙏 

1. I've just realized that a live demo link is not required in this project. was supposed to be tag as optional.
2. Making use of semantic UI too should be tag optional because the previous reviewer didn't mention it, but I will encourage you to use it.

I am sorry to mislead you. I haven't done that on purpose :blush:

**Now Let's fix the screenshot, check comment below 😄**

File: README.md [hunk 3]
```diff
@@ -1,60 +1,73 @@
...
-# Project Name
+# Portfolio: setup and mobile version skeleton
 
-> One paragraph statement about the project.
+> For the first milestone in the process of creating your portfolio website, you will:
+- set up a new repository and prepare it for development using best practices (e.g. linters).
+- build the first 3 sections of the mobile website using the template you chose in the previous step
 
-![screenshot](./app_screenshot.png)
+![screenshot](.images./printScreen.png)
```
Comment (bellom) [inline README.md]: Let's fix the screenshot.

Hint: 
I think you should double-check the name of the folder and how to set the path.
Seems you have it as `images` not `images.`

Let's try `screenshot` this should work just fine.

Comment (leonardoeng13) [reply README.md]: It was my lack of attention. My mistake. Thanks to appoint me this ".". 

The other issue about, I'm still don't understanding what's happening. When I change for the first time works, If do I follow your step-by-step, doesn't work.

Comment (bellom) [reply README.md]: @leonardoeng13 

Let me try my best to break it down, in my first comment up there, I didn't really talk about flex, I talked about Semantic HTML.

Now let me try my best to provide some hints via inline comments.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile Version</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@500&family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>   
+    <div class="wrapper">
```
Comment (bellom) [inline index.html]: Let's try to separate the sections, we have just two sessions in this project for now.

Let's use the `header` tag to wrap the `nav` tag.
Then use `display: flex` on the header tag, and you can also add `justify-content: space-between;`.

Now, do the same for the second session using flexbox.

Example:
`<header>`....
`<section>`...


@leonardoeng13 I hope this helps.

Comment (leonardoeng13) [reply index.html]: I think now it's working ... I found the readme image error ... case sensitive in the readme I wrote png and the file are PNG

Comment (leonardoeng13) [reply index.html]: I know you guys are pushing me to be better, but this 1 first week was hard for me to got myself on the tracks. Could you please approve my code?
I promise I'll review my code this weekend to improve myself, but now I'm running out of time.

**Review by ZawadiSumba66 — APPROVED** [review]
#### Hi @leonardoeng13 , 
## STATUS : APPROVED ✔️ ✔️ 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### OPTIONAL SUGGESTIONS 💡 

- [ ] Just ensure that your live demo link is actually working because now it's currently broken. However, it's not a hard requirement and thus can be removed.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**